### ISSUE TITLE
the `+package` meta may have more than one part, which is wrong

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#54]
Currently, this is possible:

```
+package hello world
```

This is wrong, since only one (and exactly one) part is expected.

**Comment by h1alexbel** [comment]
@yegor256 currently, `incorrect-package` lint will issue the following warning:

```xml
<defect line="9" severity="warning">Wrong format of package name "hello world"</defect>
```

**Comment by h1alexbel** [comment]
@yegor256 We can create a new lint that will check parts only: if `+package` contains more than 1 part, it will issue `critical`. WDYT?

**Comment by yegor256** [comment]
@h1alexbel sounds good!

**Comment by yegor256** [comment]
@h1alexbel maybe the `incorrect-package` should only check the formatting of the first `part`, not the entire `tail`? Then, the new lint will check the number of parts.

**Comment by yegor256** [comment]
@rultor release, tag is `0.0.10`
